microservices-dashboard-server/src/test/java/be/ordina/msdashboard/caching/CachingConfigTest.java

```diff
@@ -0,0 +1,66 @@
+package be.ordina.msdashboard.caching;
+
+import be.ordina.msdashboard.model.Node;
+import be.ordina.msdashboard.services.RedisService;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.runners.MockitoJUnitRunner;
+import org.springframework.data.redis.cache.RedisCacheManager;
+import org.springframework.data.redis.core.RedisTemplate;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.*;
+
+@RunWith(MockitoJUnitRunner.class)
+public class CachingConfigTest {
+
+    @InjectMocks
+    private CachingConfig cachingConfig;
+
+    @Mock
+    private CachingProperties cachingProperties;
+
+    @Test
+    @SuppressWarnings("unchecked")
+    public void testCacheManager() {
```

Comment:
Please don't use "test" as a prefix. That's for JUnit 3.
Better try "shouldDoSomething" to describe behaviour.